pretix_sepadebit/views.py

```diff
@@ -243,10 +248,24 @@ def get_queryset(self):
         ).order_by('-datetime')
 
     def get_unexported(self):
-        return OrderPayment.objects.filter(
-            order__event__organizer=self.request.organizer,
-            provider='sepadebit',
-            state=OrderPayment.PAYMENT_STATE_CONFIRMED,
-            order__testmode=False,
-            sepaexportorder__isnull=True
-        )
+        q_list = []
+        today = datetime.date.today()
+
+        for event in Event.objects.filter(
+            organizer=self.request.organizer
+        ):
+            latest_export_due_date = today + datetime.timedelta(days=int(event.settings.payment_sepadebit_prenotification_days))
```

Comment:
1. What happens here if ``payment_sepadebit_prenotification_days`` isn't set because the plugin wasn't ever enabled for an event? ``int(None)`` crashed
2. This will still cause extremely large queries for organizers with lots of events. I think we can risk it, but we should at least move the common requirements (like provider=sepadebit) out of the large OR query and put it into one common filter clause.

Reply:
I've made two changes:
1. The 'int(None)'-TypeException is caught and ignored -> exports for events which aren't set up completely are ignored
2. Added an 'event_live_issues' signal to prevent events without a set 'prenotification_days'-setting to go live

Reply:
> Added an 'event_live_issues' signal to prevent events without a set 'prenotification_days'-setting to go live

Hmm. I think the live issue should only fire if the payment provider is activated.

Reply:
Ah, I've missed the difference between enabled plugin and activated payment provider 👍